kallisto/Makefile

```diff
@@ -9,6 +9,7 @@ tag = 0.43.1--${git_commit}
 # Steps
 build:
 	docker build -t ${name}:${tag} .
+	-docker rmi -f ${name}:latest
```

Comment:
@jvivian I don't think there is supposed to be a `-f` here - at least, Toil doesn't have it. Do you want to double check that this works?